React/Server.php

```diff
@@ -11,9 +11,9 @@
  * Socket server.
  *
  * Overwrites React\Socket\Server
- * 
+ *
```

Comment:
This file is basically copied from React project. We shouldn't change unnecessary code in those files to make it easy to compare changes with origin. I hope we get rid one day of these files.